schemas/com.snowplowanalytics.snowplow.ecommerce/product/jsonschema/1-0-0

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a product entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "product",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "description" : "The SKU or product ID.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "name" : {
+            "description" : "The name or title of the product.",
+            "type": ["string", "null"],
+            "maxLength": 4096
+        },
+        "category": {
+            "description" : "The category the product belongs to. Use a consistent separator to express multiple levels. E.g. Woman/Shoes/Sneakers",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "price": {
+            "description" : "The price of the product at the current time.",
+            "type": "number",
+            "minimum": 0,
+            "multipleOf": 0.01,
+            "maximum": 2147483647
```

Comment:
So `2147483647` is a sensible maximum to use when storing a 32-bit integer, but it is not the best choice for maximum when storing a Decimal type in the warehouse.

I suggest set maximum to `9999999` and then the warehouse will use a 32-bit column to store the full range of allowed values.  Do you think `9999999` is sufficiently large as a maximum?  Or do you think we need to raise the threshold and then use a 64-bit column in the warehouse?

The same argument goes for all other places in this PR where you set `multipleOf: 0.01` and `maximum: 2147483647`.

Reply:
I think it is a sufficient number, changing it.